### DISCUSSION TITLE
Can't run docker build

### DISCUSSION THREAD

**Discussion by DimaKozhanovsky** [open-tutorials/cypress#96] [Q&A]
Colleagues, I'm doing a tutorial https://www.epic1h.com/deploy_conduit  paragraf 3.2.

When executing this the command in the title is displayed  "fg: no current job".

Please , help me to find out the reason of issue ? Thank you in advance 🙏

<img width="585" alt="Screenshot 2023-06-07 at 13 18 01" src="https://github.com/open-tutorials/cypress/assets/74324758/dbe1c456-3fed-4b03-9c7d-3f6b52e80abb">

**Comment by breslavsky** [comment]
Could you please record a video?

**Reply by DimaKozhanovsky** [reply]
The issue was closed during Stand up session , thx a lot !
